src/completion.ts

```diff
@@ -50,14 +82,24 @@ export const functions: CompletionType[] = [
     label: 'COUNTMATCHES',
     apply: snippet(`COUNTMATCHES(\${value}, \${"value"})`),
   },
-]
+  {
+    label: 'ADDDAYS',
+    apply: snippet(`ADDDAYS(\${value}, \${2})`),
+  },
+  {
+    label: 'DAYSBETWEEN',
+    apply: snippet(`DAYSBETWEEN(\${value}, \${value})`),
```

Comment:
i think this add two cursors, i cant find a good doc for this :/
```suggestion
    apply: snippet(`DAYSBETWEEN(\${value}, value)`),
```